packages/manager/apps/dedicated/client/app/license/translations/Messages_fr_FR.json

```diff
@@ -324,6 +324,16 @@
   "license_designation_WINDOWS_version_WINDOWS_SERVER_2016_DATACENTER_EDITION_1_CPU_10_CORES": "Microsoft Windows Server 2016 Datacenter Edition (1 CPU 10 CORES)",
   "license_designation_WINDOWS_version_windows-server-2016-license-datacenter-edition-1-cpu-10-cores": "Microsoft Windows Server 2016 Datacenter Edition (1 CPU 10 CORES)",
   "license_designation_WINDOWS_version_WINDOWS_SERVER_2016_DATACENTER_EDITION_1_CPU_12_CORES": "Microsoft Windows Server 2016 Datacenter Edition (1 CPU 12 CORES)",
+  "license_designation_WINDOWS_version_WINDOWS_SERVER_2016_DATACENTER_EDITION_1_CPU_14_CORES": "MICROSOFT Windows Server 2016 Datacenter edition (1 CPU - 14 CORES)",
```

Comment:
HI Mari, Need to do analysis.

Reply:
Any news? We're injecting a lot of content here, where we could only parse the key and extract specific information

Reply:
Hello Jeremy, Its quit complicated as its not a direct change we need to do modification is several places. Since this ticket is critical we need to send it to prod now. We can add new ticket to do the new parsing.

Reply:
Seen together